charts/dispatch/values.yaml

```diff
@@ -25,5 +25,8 @@ global:
     database: dispatch
     namespace: dispatch
     release: postgres
+  registry:
+    insecure: true
```

Comment:
I prefer the default to be *insecure: false* otherwise even valid registries will be marked as insecure in dind configuration. We can expose this as a config option in DispatchConfig.ImageRegistry and let the user control it. I do understand this will be an additional config for someone doing *helm install stable/docker-registry*.